### PR TITLE
More custom border styles.

### PR DESCRIPTION
**Pull request by pnowak**
### Context
Allowing border styling like `dotted`, `dashed` or `double`. And animation these if needed.

### How has this been tested?
Add `style` property to concrete properties of `borderObject` - eg. ```left: {
            width: 3,
            color: 'red',
            style: 'dotted'
          },```.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #5245 
2. #5568 

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/3rdparty/walkontable/src/border.js [hunk 1]
```diff
@@ -616,7 +680,9 @@ class Border {
         removeClass(this[borderElement], 'hidden');
       }
 
-      style.backgroundColor = borderStyle.color;
+      if (hasClass(this[borderElement], `htAnimateCustomBorders${toUpperCaseFirst(borderElement)}`)) {
+        removeClass(this[borderElement], `htAnimateCustomBorders${toUpperCaseFirst(borderElement)}`);
+      }
```
Comment (swistach) [inline src/3rdparty/walkontable/src/border.js]: Could you remove this `if`? It's no longer necessary :)

File: src/3rdparty/walkontable/src/border.js [hunk 2]
```diff
@@ -441,18 +448,72 @@ class Border {
...
     this.bottomStyle.width = `${width}px`;
     this.bottomStyle.display = 'block';
 
     this.rightStyle.top = `${top}px`;
-    this.rightStyle.left = `${left + width - delta}px`;
+    this.rightStyle.left = `${left + width - delta - extra}px`;
     this.rightStyle.height = `${height + 1}px`;
     this.rightStyle.display = 'block';
 
+    if (this.settings.right && this.settings.right.hide !== true) {
```
Comment (swistach) [inline src/3rdparty/walkontable/src/border.js]: I'm not sure this delta calculations works properly. If I use `width: 10px` then borders looks strange.
screenshot 2019-02-14 at 10 45 09

Comment (pnowak) [reply src/3rdparty/walkontable/src/border.js]: In pro version is ok.

File: src/3rdparty/walkontable/src/border.js (hunk 1 above)
Comment (pnowak) [reply src/3rdparty/walkontable/src/border.js]: Removed.

**Comment by pnowak** [comment]
As I wrote in the issue we need to rethink the way we're adding `customBorders`.

**Pull request closed without merging**